Black-Market/Black-Market/Extensions/ColorExtension.swift

```diff
@@ -10,4 +10,5 @@ extension Color {
     static let blackStyle = Color("BlackStyle")
     static let lightGrey = Color("LightGreyStyle")
     static let darkGrayStyle = Color("DarkGrayStyle")
+    static let blueStyle = Color("BlueStyle")
```

Comment:
Is there any reason to add the `Style` suffix to colors?

Reply:
Removed